artifacts/rhdh-plugins-reference/argocd/argocd-plugin-admin.adoc

```diff
@@ -70,6 +70,143 @@ global:
         disabled: false
 ----
 
+== (Optional) Enabling Argo CD Rollouts feature
+
+The Argo CD Rollouts feature enhances Kubernetes by providing advanced deployment strategies, such as blue-green and canary deployments, for your applications. When integrated into the backstage Kubernetes plugin, it allows developers and operations team to visualize and manage these Rollouts seamlessly within the Backstage interface.
+
+.Prerequisites
+
+* *Backstage Kubernetes plugin installed:* Verify that the @backstage/plugin-kubernetes-backend is installed and configured. Follow the installation guide and configuration guide if needed.
+
+* *Kubernetes cluster access:* Ensure you have access to the Kubernetes cluster with the necessary permissions to create and manage custom resources and `ClusterRoles`.
+
+* *Argo Rollouts resources:* Confirm that the Kubernetes cluster has the `argoproj.io` group resources (for example, Rollouts and AnalysisRuns) installed.
+
+.Procedures
+
+. Update `app-config.yaml` for custom resources.
+
+.. Open the `app-config.yaml` file in your Backstage instance.
+
+.. Add the following `customResources` component under the `kubernetes` configuration to enable Argo Rollouts and AnalysisRuns:
++
+[source,yaml]
+----
+kubernetes:
+  ...
+  customResources:
+    - group: 'argoproj.io'
+      apiVersion: 'v1alpha1'
+      plural: 'Rollouts'
+    - group: 'argoproj.io'
+      apiVersion: 'v1alpha1'
+      plural: 'analysisruns'
+----
+
+. Grant `ClusterRole` permissions for custom resources.
+
+.. Verify whether the Backstage Kubernetes plugin is already configured. If it is, the `ClusterRole` for Rollouts and AnalysisRuns might already be granted.
++
+[NOTE]
+====
+You can use a prepared manifest for a read-only `ClusterRole` that provides access for both the Kubernetes plugin and the ArgoCD plugin.
```

Comment:
> prepared manifest for a read-only `ClusterRole`

the above text should be a link - https://raw.githubusercontent.com/backstage/community-plugins/main/workspaces/redhat-argocd/plugins/argocd/manifests/clusterrole.yaml